models/public/hrnet-w32-human-pose-estimation/hrnet-w32-human-pose-estimation.md

```diff
@@ -0,0 +1,85 @@
+# hrnet-w32-human-pose-estimation
+
+## Use Case and High-Level Description
+
+The `HRNet-W32` model is one of the [HigherHRNet](https://arxiv.org/pdf/1908.10357).
+`HigherHRNet` is a novel bottom-up human pose
+estimation method for learning scale-aware representations using high-resolution feature pyramids. The network uses HRNet as backbone, followed by one or more deconvolution modules to generate multi-resolution and high-resolution heatmaps. For every person in an image, the network detects a human pose: a body skeleton consisting of keypoints and connections between them. The pose may contain up to 17 keypoints: ears, eyes, nose, shoulders, elbows, wrists, hips, knees, and ankles.
+This is PyTorch\* implementation pretrained on COCO dataset.
+For details about implementation of model, check out the [HigherHRNet: Scale-Aware Representation Learning for Bottom-Up Human Pose Estimation](https://github.com/HRNet/HigherHRNet-Human-Pose-Estimation) repository.
+
+## Specification
+
+| Metric            | Value                  |
+|-------------------|------------------------|
+| Type              | Human pose estimation  |
+| GFLOPs            | 92.8364                |
+| MParams           | 28.6180                |
+| Source framework  | PyTorch\*              |
+
+## Accuracy
+
+| Metric | Original model | Converted model |
+| ------ | -------------- | --------------- |
+| Average Precision (AP)  | 65,02%          | 65,02%           |
+
+## Input
+
+### Original Model
+
+Image, name - `image`,  shape - `1,3,512,512`, format is `B,C,H,W`, where:
+
+- `B` - batch size
+- `C` - channel
+- `H` - height
+- `W` - width
+
+Channel order is `RGB`. Mean values - [123.675,116.28,103.53], scale values - [58.395,57.12,57.375].
+
+### Converted Model
+
+Image, name - `image`,  shape - `1,3,512,512`, format is `B,C,H,W`, where:
+
+- `B` - batch size
+- `C` - channel
+- `H` - height
+- `W` - width
+
+Channel order is `BGR`.
+
+## Output
+
+The net outputs three blobs:
+
+- `heatmaps` of shape B, 17, 256, 256 containing location heatmaps for keypoints of all types.
```

Comment:
```suggestion
- `heatmaps` of shape `B, 17, 256, 256` containing location heatmaps for keypoints of all types.
```

Reply:
what do you mean under all types?

Reply:
B is batch size, right? If you have fixed batch size in input shape, I believe it does not change in output, so 1 should be instead of B